nodes-media/Dockerfile

```diff
@@ -25,4 +28,4 @@ RUN yarn build
 # server api
 EXPOSE 5454
 
-CMD [ "yarn", "start" ]
\ No newline at end of file
+CMD [ "yarn", "dev" ]
```

Comment:
should not do yarn dev in a production deployment -- this will be deployed in production

Reply:
switch this back to yarn start -- if the issue was watching the filesystem for changes for local dev, then we need to change the dev script